charts/govuk-jobs/templates/content-store-mongo-to-postgres-cronjob.yaml

```diff
@@ -174,9 +174,9 @@ spec:
               image: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/github-cli:latest
               command: ["pg_dump"]
               args:
-                - "-F c"
```

Comment:
I suspect it wanted:

```
- "-F"
-  c
- "-f"
... etc
```

I also don't think the `$(date ...)` expansion is going to happen, because this array-style argument format makes me think there's not going to be a shell involved.

You might need to consider doing:

```
command: ["bash"]
args:
  - "-c"
  - "pgdump -F c -f /pg-dump/.... ...." 
  ```

Reply:
Thanks, I'll try that next (assuming this one doesn't work :) )

Reply:
Tried it in #1253 1253